### PR TITLE
GHO Facilitator and Cap Updates

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Update facilitator capacity.
Update Arbitrum caps.

### CODE REVIEW

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: "[AIP] Arbitrum - GHO Stewards & Parameter Adjustments"
+author: "@karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-gho-arbitrum-parameter-adjustments/18386"
+snapshot: "Direct-to-AIP"
+---
+
+## Simple Summary
+
+This AIP will activate the GHO Steward and adjust several GHO related parameters on Arbitrum.
```
Comment (brotherlymite) [inline src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md:10]: Stewards from the AIP text should be removed here

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: "[AIP] Arbitrum - GHO Stewards & Parameter Adjustments"
```
Comment (brotherlymite) [inline src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md:2]: We should remove `GHO Stewards` from the title too

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/AaveV3Arbitrum_IncreaseGHOFacilitatorCapacity_20240722.t.sol [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      'AaveV3Arbitrum_IncreaseGHOFacilitatorCapacity_20240722',
+      AaveV3Arbitrum.POOL,
+      address(proposal)
+    );
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_newLimitIsSet() public {
```
[the remark this answers is not in the document]
Comment (efecarranza) [reply src/20240722_Multi_IncreaseGHOFacilitatorCapacity/AaveV3Arbitrum_IncreaseGHOFacilitatorCapacity_20240722.t.sol:41]: @pavelvm5 @brotherlymite doesn't the `defaultTest` cover these?
thank you

[the remark this answers is not in the document]
Comment (brotherlymite) [reply src/20240722_Multi_IncreaseGHOFacilitatorCapacity/AaveV3Arbitrum_IncreaseGHOFacilitatorCapacity_20240722.t.sol:40]: `defaultTest` does e2e test actions like deposit, borrow, withdraw... and for caps it only outputs the diffs

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
+---
+title: "[AIP] Arbitrum - Parameter Adjustments"
```
Comment (brotherlymite) [inline src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md:2]:
```suggestion
title: "GHO Arbitrum Parameter Adjustments"
```

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+The parameter adjustments presented below are endorsed by Chaos Labs as an Aave DAO Risk Service Provider.
+
+## Specification
+
+This proposal will implement the following parameter adjustments.
+
+| Parameter                    | Current | Proposed |
+| ---------------------------- | :-----: | :------: |
+| Eth GHO Bucket Capacity      |  2.5M   |  20.0M   |
```
Comment (brotherlymite) [inline src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity.md:26]:
```suggestion
| Eth GHO Bridge Limit      |  2.5M   |  20.0M   |
```

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/AaveV3Arbitrum_IncreaseGHOFacilitatorCapacity_20240722.sol [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+import {MiscArbitrum} from 'aave-address-book/MiscArbitrum.sol';
+import {IGhoToken} from 'gho-core/gho/interfaces/IGhoToken.sol';
+import {AaveV3PayloadArbitrum} from 'aave-helpers/v3-config-engine/AaveV3PayloadArbitrum.sol';
+import {IAaveV3ConfigEngine} from 'aave-v3-origin/periphery/contracts/v3-config-engine/IAaveV3ConfigEngine.sol';
+
+/**
+ * @title Increase GHO Facilitator Capacity
+ * @author @karpatkey_TokenLogic
+ * - Snapshot: Direct-to-AIP
+ * - Discussion: https://governance.aave.com/t/arfc-gho-arb-increase-ccip-facilitator-capacity/18169
```
Comment (pavelvm5) [inline src/20240722_Multi_IncreaseGHOFacilitatorCapacity/AaveV3Arbitrum_IncreaseGHOFacilitatorCapacity_20240722.sol:14]: Could you change discussion link here too?

File: .assets/62fc3d2ec2423de7df4fe3e6db7f333bec8e7554.svg
Comment (pavelvm5) [inline .assets/62fc3d2ec2423de7df4fe3e6db7f333bec8e7554.svg:1]: three svg files are still in Pr.
.assets/62fc3d2ec2423de7df4fe3e6db7f333bec8e7554.svg
.assets/663480b8122e6db82476beb1a6c8b1d576e9b7b6.svg
.assets/6b6740b86ac103570c299958558e21a71231fc00.svg

File: src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity_20240722.s.sol [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+/**
+ * @dev Create Proposal
+ * command: make deploy-ledger contract=src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity_20240722.s.sol:CreateProposal chain=mainnet
+ */
+contract CreateProposal is EthereumScript {
+  function run() external {
+    // create payloads
+    PayloadsControllerUtils.Payload[] memory payloads = new PayloadsControllerUtils.Payload[](3);
```
Comment (brotherlymite) [inline src/20240722_Multi_IncreaseGHOFacilitatorCapacity/IncreaseGHOFacilitatorCapacity_20240722.s.sol:61]:
```suggestion
    PayloadsControllerUtils.Payload[] memory payloads = new PayloadsControllerUtils.Payload[](2);
```

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**